docs/types.md

```diff
@@ -1045,6 +1045,23 @@ Below are the currently supported file operations:
 | `make-dir` | `mkdir`    | path                   | `mkdir -p`       | Create a directory at the given path. The path make already exist. Any required parent directories will be created. |
 | `remove`   | `rm`       | path                   | `rm -rf`         | Remove the given path. The path may be a file or directory. The path may not exist. |
 
+Any `move` or `copy` command may include globs in the source path.
+We support the [Go standard for matching globs](https://pkg.go.dev/path/filepath#Match).
+We use lazy evaluation for globs, so we reduce the source path before expanding globs.
```

Comment:
The "lazy" part is not exactly correct, since they are cannot be evaluated when they are first parsed.
I would be more direct and say that paths with globs simplified as if the globs were normal names before they are expanded.